### DISCUSSION TITLE
any chance to run jacktrip on android?

### DISCUSSION THREAD

**Discussion by recotra** [jacktrip/jacktrip#864] [General]
My use scenario is to use my android phone as mic input and sound output to my other windows desktop, thank you!

**Comment by psiborg112** [comment]
As far as I'm aware, there aren't currently any plans to port JackTrip to mobile platforms. (At least not on the open source side of development.) I'm not sure exactly what your use case is, but if you're just wanting to use your phone as a microphone, something like [droidcam](https://www.dev47apps.com/) might be worth checking out. (It allows you to use your android device as a mic and webcam.) Don't know of anything offhand for bidirectional audio flow.
